ParabolInc/action
# feat(Azure DevOps): Adding additional project types

# Description

Fixes/Partially Fixes #6512 
This includes a mapping of app out-of-the-box Azure DevOps Project type work items to their default fields for sprint planning purposes.

## Testing scenarios

Add import work items for different Azure DevOps project types such as (Agile, Scrum, Basic).

- [ ] Agile
  - Verify an Epic work item can update the Effort field (or comment) in Azure DevOps during a sprint poker meeting
  - Verify a Feature can update the Effort field (or comment) in Azure DevOps during a sprint poker meeting
  - Verify the User Story can update the StoryPoints (or comment) in Azure DevOps during a sprint poker meeting
  - Verify the Task can update the OriginalEstimate (or comment) in Azure DevOps during a sprint poker meeting

- [ ]Basic
  - Verify the Issue can update the Effort field (or comment) in Azure DevOps during a sprint planning meeting 
  - Verify the Task can update the RemainingWork (or comment) field in Azure DevOps during a sprint poker meeting

- [ ]Scrum
  - Verify the Bug can update the Effort field (or comment) in Azure DevOps during a sprint planning meeting 
  - Verify the Epic can update the Effort (or comment) field in Azure DevOps during a sprint poker meeting
  - Verify the Feature can update the Effort (or comment) field in Azure DevOps during a sprint poker meeting
  - Verify the Product Backlog Item can update the Effort (or comment) field in Azure DevOps during a sprint poker meeting
  - Verify the Task can update the Remaining Work (or comment) field in Azure DevOps during a sprint poker meeting

- [ ]CMMI
  - Verify the Bug can update the Size field (or comment) in Azure DevOps during a sprint planning meeting 
  - Verify the Change Request can update the Effort (or comment) field in Azure DevOps during a sprint poker meeting
  - Verify the Epic can update the Effort (or comment) field in Azure DevOps during a sprint poker meeting
  - Verify the Feature can update the Effort (or comment) field in Azure DevOps during a sprint poker meeting
  - Verify the Issue can update the OriginalEstimate (or comment) field in Azure DevOps during a sprint poker meeting
  - Verify the Requirement can update the Size (or comment) field in Azure DevOps during a sprint poker meeting
  - Verify the Risk can update the OriginalEstimate (or comment) field in Azure DevOps during a sprint poker meeting
  - Verify the Task can update the OriginalEstimate (or comment) field in Azure DevOps during a sprint poker meeting
